addons/vehiclelock/CfgMagazines.hpp

```diff
@@ -5,7 +5,7 @@ class CfgMagazines {
         displayName = "ACE Vehicle Key";   //!!!CANNOT be localized!!!: because it is used as part of the magazineDetail string
         descriptionShort = CSTRING(Item_Custom_Description);
         count = 1;
-        mass = 0;
+        mass = 0.1;
```

Comment:
```suggestion
        mass = 0;
```
these are scope=1 and are really only going to be added via script so it should be fine to keep at 0
the reason I set them to mass=0 is so the script that adds them could never fail to do `addMagazine`
e.g. player's loadout is full, you can still add a mass=0 mag

Reply:
Can't you just use container addMagazineCargoGlobal instead? Never fails, even when full. You need to figure out the container though, but that doesn't sound hard.

Reply:
I increased their weight so they match with the other item. I don't mind undoing it.